aperturedb/Connector.py

```diff
@@ -396,7 +396,8 @@ def _connect(self):
                         f"The host name must match the certificate: {self.host} " + os.linesep + \
                         f"You can use the ca_cert parameter to specify a custom CA certificate " + os.linesep + \
                         f"Refer to the documentation for more information: {SETUP_URL}" + os.linesep + \
-                        f"Alternatively, SSL can be disabled by setting use_ssl=False (not recommended)"
+                        f"Alternatively, SSL can be disabled by setting use_ssl=False (not recommended)" + os.linesep + \
+                        f"{e=}"
                 except ssl.SSLError as e:
                     logger.error(f"Error wrapping socket: {e}")
                     self.conn.close()
```

Comment:
I don't think it's necessary to close the connection here, given the `BaseException` catch below.